### PR TITLE
#657 - Updates as per the comments and changes to the test function

### PR DESCRIPTION
**Pull request by HarshSharma12**
# Before submitting

- [ ] Was this discussed/approved via a Github issue? (no need for typos, doc improvements)
- [ ] Did you read the [contributor guideline](https://github.com/PyTorchLightning/pytorch-lightning/blob/master/.github/CONTRIBUTING.md)?
- [ ] Did you make sure to update the docs?   
- [ ] Did you write any new necessary tests?  

## What does this PR do?
Fixes # 657. 
Updates as per the comments (https://github.com/PyTorchLightning/pytorch-lightning/pull/723) and changes to the test function

## PR review    
Anyone in the community is free to review the PR once the tests have passed.     
If we didn't discuss your PR in Github issues there's a high chance it will not be merged.

## Did you have fun?
Make sure you had fun coding 🙃

### CODE REVIEW

File: tests/utils.py [hunk 1]
```diff
@@ -65,6 +65,10 @@ def run_model_test(trainer_options, model, on_gpu=True):
     # correct result and ok accuracy
     assert result == 1, 'amp + ddp model failed to complete'
 
+    if (early_stop):
```
Comment (Borda) [inline tests/utils.py]: why the brackets?

Comment (HarshSharma12) [reply tests/utils.py]: just a habit. resolved now

**Comment by Borda** [comment]
pls edit the description # 657 -> #657

**Comment by williamFalcon** [comment]
@HarshSharma12 rebase?

**Comment by HarshSharma12** [comment]
PR# 737 overrides this. All comments addressed there. Rebase was not helping in resolving git conflicts hence closing this. Apologies for the confusion.

**Pull request closed without merging**